contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/HadoopStoreBuilder.java

```diff
@@ -348,11 +372,9 @@ public void build() {
                 Path directoryPath = new Path(outputDir.toString(), directoryName);
 
                 if(!outputFs.exists(directoryPath)) {
-                    logger.info("No data generated for " + directoryName
-                                        + ". Generating empty folder");
+                    emptyDirectories.add(directorySuffix);
                     outputFs.mkdirs(directoryPath); // Create empty folder
                     outputFs.setPermission(directoryPath, new FsPermission(HADOOP_FILE_PERMISSION));
-                    logger.info("Setting permission to 755 for " + directoryPath);
```

Comment:
Can we keep this as a debug statement? Perhaps merged into the other statement above, if you'd like.